### PR TITLE
halo projectors, emissives update, accessories update, etc

### PR DESCRIPTION
**Pull request by TrojanCoyote**
adds a halo projector which projects... you guessed it, halos!

## Changelog

:cl:
add: Added more things
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -403,6 +403,154 @@
...
+/obj/item/clothing/accessory/haloprojector/proc/reskin_halo_projector(mob/living/L)
+	var/obj/item/halo_projector
+	var/list/halo_projector_list = subtypesof(/obj/item/clothing/accessory/haloprojector)
+	var/list/display_names = list()
+	var/list/halo_projector_icons = list()
+	for(var/V in halo_projector_list)
+		var/obj/item/clothing/accessory/haloprojector/halotype = V
+		if (V)
+			display_names[initial(halotype.name)] = halotype
+			halo_projector_icons += list(initial(halotype.name) = image(icon = initial(halotype.icon), icon_state = initial(halotype.icon_state)))
```
Comment (Kapu1178) [inline code/modules/clothing/under/accessories/accessory.dm:430]:
```suggestion
	for(var/obj/item/clothing/accessory/haloprojector/halotype as anything in halo_projector_list)
		var/obj/item/clothing/accessory/haloprojector/halotype = V
		halo_projector_icons += list(initial(halotype.name) = image(icon = initial(halotype.icon), icon_state = initial(halotype.icon_state)))
```

**Comment by silicons** [comment]
this pr is obviously now also my PR because i'm actively refactoring inside it lmao

File: code/__HELPERS/verbs.dm [hunk 2]
```diff
@@ -108,7 +108,7 @@
  * makes replacing easier later
  */
 /proc/add_obj_verb(atom/A, verb_or_list_to_add)
-	A.verbs += verb_or_list_to_add
+	A.verbs |= verb_or_list_to_add
```
Comment (TheLordME) [inline code/__HELPERS/verbs.dm:111]: LAZYDISTINCTADD?

**Review by TheLordME — APPROVED** [review]

File: code/__HELPERS/verbs.dm (hunk 2 above)
Comment (silicons) [reply code/__HELPERS/verbs.dm:111]: verbs is internal byond list, not a lazylist

**Pull request merged**